src/foraging_gui/Foraging.py

```diff
@@ -241,6 +247,7 @@ def connectSignalsSlots(self):
         self.actionConnectBonsai.triggered.connect(self._ConnectBonsai)
         self.actionReconnect_bonsai.triggered.connect(self._ReconnectBonsai)
         self.Load.clicked.connect(self._OpenLast)
+        self.Save.setCheckable(True)
```

Comment:
It allows the Save button to be set to checked when pressed by a user. I needed the save button to be checkable so I could tell if the _Save function was called by a user pressing the save button (Save.isChecked() == True) or if it was called by somewhere else in the code. I think some of the other buttons like Start are set checkable somewhere in the ui files since Start.isChecked() is used throughout the code, but I'm assuming the Save button is not because I couldn't get it to stay checked without adding this line.